examples/servers/code_actions.py

```diff
@@ -67,6 +67,7 @@ def code_actions(params: types.CodeActionParams):
                 title=f"Evaluate '{match.group(0)}'",
                 kind=types.CodeActionKind.QuickFix,
                 edit=types.WorkspaceEdit(changes={document_uri: [text_edit]}),
+                tags=[types.CodeActionTag.LlmGenerated],
```

Comment:
Perhaps we could drop this? It's nice to be able to show the new features of `lsprotocol` but I wonder if it might confuse someone new to language servers into thinking it's required?

Reply:
Yes we can. I will make the update.